Observer/CaptureObserver.php

```diff
@@ -315,7 +321,7 @@ private function sendInvoiceRequest($invoice, $orderLines, $orderObject, $paymen
     }
 
     /**
-     * @param $order
+     * @param null|object $invoice
      *
      * @return float|int
```

Comment:
```suggestion
     * @param Magento\Sales\Model\Order\Invoice $invoice
     *
     * @return float
```

Maybe also change `$weeTaxAmount = 0;` to `$weeTaxAmount = 0.0;`